java/ql/src/experimental/Security/CWE/CWE-094/InsecureDexLoading.qhelp

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>
+Shared world writable storage spaces are not secure to load Dex libraries from. A malicious actor can replace a dex file with a maliciously crafted file 
+which when loaded by the app can lead to code execution. 
+</p>
+  </overview>
+
+  <recommendation>
+    <p>
+      Loading a file from private storage instead of a world writable one can prevent this issue.
+      As the attacker cannot access files stored by the app in its private storage. 
```

Comment:
```suggestion
      Loading a file from private storage instead of a world-writable one can prevent this issue,
      because the attacker cannot access files stored there. 
```